pkg/controllers/v1alpha1/dataload/implement.go

```diff
@@ -118,7 +118,38 @@ func (r *DataLoadReconcilerImplement) reconcileNoneDataLoad(ctx reconcileRequest
 func (r *DataLoadReconcilerImplement) reconcilePendingDataLoad(ctx reconcileRequestContext) (ctrl.Result, error) {
 	log := ctx.Log.WithName("reconcilePendingDataLoad")
 
-	// 1. Check existence of the target dataset
+	// 1. Check dataload namespace and dataset namespace need to be same
+	if ctx.DataLoad.Namespace != ctx.DataLoad.Spec.Dataset.Namespace {
+		r.Recorder.Eventf(&ctx.DataLoad,
+			v1.EventTypeNormal,
```

Comment:
I think it better to change `EventTypeNormal` to `EventTypeWarning` since it will make DataLoad fail fast.

Reply:
tks, done.